<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recipe_Engineer_Viewer</class>
 <widget class="QWidget" name="Recipe_Engineer_Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>647</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1240</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1240</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="topMargin">
    <number>22</number>
   </property>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupbox_ingredient_information">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>220</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Ingredient Information</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_amount_used">
           <property name="font">
            <font>
             <pointsize>33</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="viz_treemap_legend"/>
      </item>
      <item>
       <layout class="QGridLayout" name="layout_ingredient_grid_info_text">
        <item row="0" column="1" alignment="Qt::AlignLeft|Qt::AlignTop">
         <widget class="QLabel" name="output_ingredient_category">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1" alignment="Qt::AlignLeft|Qt::AlignTop">
         <widget class="QLabel" name="output_ingredient_brand">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignTop">
         <widget class="QLabel" name="label_ingredient_category">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Category:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" alignment="Qt::AlignLeft|Qt::AlignTop">
         <widget class="QLabel" name="output_ingredient_supplier">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" alignment="Qt::AlignTop">
         <widget class="QLabel" name="label_ingredient_name">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" alignment="Qt::AlignTop">
         <widget class="QLabel" name="label_ingredient_supplier">
          <property name="text">
           <string>Supplier:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_ingredient_price">
          <property name="text">
           <string>Price:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" alignment="Qt::AlignTop">
         <widget class="QLabel" name="label_ingredient_brand">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Brand:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="output_ingredient_number">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="output_ingredient_price">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="output_ingredient_name">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_ingredient_number">
          <property name="text">
           <string>Number:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Volatility:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="label_volatility_ingredient_info">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QPushButton" name="btn_add_ingredient">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupbox_recipe_engineering">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Recipe Composition</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="cbox_category"/>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="layout_horizontal_filter">
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_ingredient_filter">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="input_ingredient_filter">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="input_recipe_name"/>
      </item>
      <item row="1" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_recipe_name">
        <property name="text">
         <string>Recipe Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_category">
        <property name="text">
         <string>Recipe Category</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QListView" name="list_select_ingredients"/>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="layout_vertical_1">
        <item alignment="Qt::AlignRight|Qt::AlignTop">
         <widget class="QLabel" name="label_ingredients">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Ingredients</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="layout_grid_1">
          <item row="2" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
           <widget class="QLabel" name="label_sort_type">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>↑↓ Category</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" alignment="Qt::AlignVCenter">
           <widget class="QRadioButton" name="radio_sort_type">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
           <widget class="QLabel" name="label_sort_name">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>↑↓ Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" alignment="Qt::AlignVCenter">
           <widget class="QRadioButton" name="radio_sort_name">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
           <widget class="QLabel" name="label_sort_price">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>↑↓ Price</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" alignment="Qt::AlignVCenter">
           <widget class="QRadioButton" name="radio_sort_price">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2" rowspan="3">
    <widget class="QGroupBox" name="groupbox_recipe_information">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>520</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Recipe Specification</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="layout_recipe_ingredients"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="layout_recipe_info_text">
          <item row="1" column="1" alignment="Qt::AlignRight|Qt::AlignBottom">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>$</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" alignment="Qt::AlignBottom">
           <widget class="QLabel" name="label_recipe_price">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Recipe Price:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" alignment="Qt::AlignLeft|Qt::AlignBottom">
           <widget class="QLabel" name="label_out_recipe_cost">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" alignment="Qt::AlignRight|Qt::AlignBottom">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>%</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignBottom">
           <widget class="QLabel" name="label_recipe_cost">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Recipe Cost:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" alignment="Qt::AlignBottom">
           <widget class="QLabel" name="label_recipe_volatility">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Recipe Volatility:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" alignment="Qt::AlignLeft|Qt::AlignBottom">
           <widget class="QLineEdit" name="input_recipe_price">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2" alignment="Qt::AlignLeft|Qt::AlignBottom">
           <widget class="QLabel" name="label_out_recipe_volatility">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" alignment="Qt::AlignRight|Qt::AlignBottom">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>$</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="btn_save_recipe">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCustomPlot" name="viz_recipe_cost" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
